synapse/config/experimental.py

```diff
@@ -169,6 +169,11 @@ def read_config(self, config: JsonDict, **kwargs: Any) -> None:
         # MSC3925: do not replace events with their edits
         self.msc3925_inhibit_edit = experimental.get("msc3925_inhibit_edit", False)
 
+        # MSC3873: Disambiguate event_match keys.
+        self.msc3783_escape_event_match_key = experimental.get(
+            "msc3783_escape_event_match_key", False
```

Comment:
Shouldn't this be this?

```suggestion
        self.msc3783_escape_event_match_key = experimental.get(
            "msc3783_escape_event_match_key", False
```

Reply:
Sorry -- I don't see any change in your suggestion?

Reply:
😅 did the same typo. Now corrected.

Reply:
Oh the MSC # is typoed, I see. Umm, yeah that's not awesome, although not super important. 😢

Reply:
See #15138